kernel/typing.mli

```diff
@@ -53,3 +53,5 @@ val inference   : Signature.t -> term -> typ
 
 val check_rule  : Signature.t -> untyped_rule -> Subst.Subst.t * typed_rule
 (** [check_rule sg ru] checks that a rule is well-typed. *)
+
+val typed_rule_of_ri : Signature.t -> rule_infos -> typed_rule
```

Comment:
We should keep the long name "rule_infos", otherwise we should be consistent and rename other functions related to `rule_infos`.